include/pcl_conversions/pcl_conversions.h

```diff
@@ -77,27 +76,28 @@ namespace pcl_conversions {
   /** PCLHeader <=> Header **/
 
   inline
-  void fromPCL(const pcl::uint64_t &pcl_stamp, ros::Time &stamp)
+  void fromPCL(const pcl::uint64_t &pcl_stamp, builtin_interfaces::msg::Time &stamp)
   {
-    stamp.fromNSec(pcl_stamp * 1000ull);  // Convert from us to ns
+    stamp.sec = (int32_t)(pcl_stamp / 1000000ull); // Convert whole seconds from us to sec
```

Comment:
Actually, it turns out that there is a good reason *not* to use the scientific notation.  When you specify something like "1e9", it is implicitly a double.  In the cases above, I actually have to have the unsigned long (otherwise I get compilation errors).  So either we use the long form 1000000ull (yuck) or we use the scientific notation with a cast (unsigned long long)1e9 (double yuck).  I opted to stay with the original long form, but let me know if you think differently.